lib/commentOnOpenedPullRequest.js

```diff
@@ -80,13 +81,15 @@ commentOnOpenedPullRequest._implementation = function (pullRequestFilesUrl, pull
         });
 };
 
-commentOnOpenedPullRequest._askAboutChanges = function (files) {
+commentOnOpenedPullRequest._askAboutChanges = function (files, baseBranch) {
+    var changesMDWasUpdated = false;
```

Comment:
I think we don't need to run the check to see if `CHANGES.MD` has been updated at all. I think just check `baseBranch === 'master'`, and if so, return false before running the file tests.

Reply:
Good catch! I updated it.